src/grocker/builders.py

```diff
@@ -260,13 +251,15 @@ def docker_get_client(**kwargs):
     all_extra_kwargs = docker.utils.kwargs_from_env(**extra_kwargs)
     all_extra_kwargs.update(kwargs)
     all_extra_kwargs['version'] = DOCKER_API_VERSION
-    client = docker.Client(**all_extra_kwargs)
+    client = docker.DockerClient(**all_extra_kwargs)
     client.version()  # Call the API, this will raise if API version is not compatible.
     return client
 
 
 def docker_build_image(docker_client, path, tag=None, pull=True, buildargs=None):
-    stream = docker_client.build(path=path, tag=tag, rm=True, forcerm=True, pull=pull, buildargs=buildargs)
+    stream = docker_client.images.client.api.build(
```

Comment:
You could also add `decode=True` to auto-magically get decoded dicts and avoid the ugly `docker.utils.json_stream.json_stream` in `inspect_stream`